set.mm

```diff
@@ -530526,7 +530530,7 @@ automatically a set (see ~ bj-mooreset ).  (Contributed by BJ,
 
```

Comment:
Why did you change the order of theses definitions? I think it is usual to first declare (all) new constant symbols (within a section), then to define their types and then provide the corresponding definitions. See, for example, section "Relations"

Reply:
This can be changed, but I thought it was preferred to have "token-syntax-definition" as three consecutive statements for most symbols.  For sets of symbols which only make sense as a whole, one may order them as you wrote.  See #2404 (where the general case is not settled); for the moment it's more on a case-by-case basis.

Reply:
If the definitions are placed in separate (sub)sections, as done in #2404, then it is OK. But curry_ and uncurry_ belong together, i.e. are a "set of (related) symbols", so they would remain in the same (sub) section.